pandas/_libs/tslibs/conversion.pyx

```diff
@@ -31,6 +30,7 @@ from pandas._libs.tslibs.util cimport (
 
 from pandas._libs.tslibs.timedeltas cimport (cast_from_unit,
                                              delta_to_nanoseconds)
+from pandas._libs.tslibs.timestamps cimport _Timestamp
```

Comment:
This is a circular import, should be avoided like the plague.

The way to do this is to separate `_Timestamp` out into a separate file that doesn't depend on `conversion`

Reply:
Ah ok, I was under the mistaken impression that circulars were ok in cython `cimport`, and was trying to avoid major refactoring.  I'll bite the bullet and move it out then.